documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -154,7 +154,7 @@ Pi 4B and Pi 400 (not CM4 or CM4IO) add a high performance USB 3 controller, att
 
 `otg_mode=1` requests that a more capable XHCI USB 2 controller is used as another host controller on that USB-C connector. 
 
-NOTE: Because CM4 and CM4IO don't include the external USB 3 controller RaspberryPi OS images set `otg_mode=1` on CM4 for better performance.
+NOTE: Because CM4 and CM4IO don't include the external USB 3 controller Raspberry Pi OS images set `otg_mode=1` on CM4 for better performance.
```

Comment:
Should probably be a comma after "controller".

Reply:
@lurch 
Thanks, done.